python/ray/serve/_private/replica.py

```diff
@@ -494,95 +515,74 @@ def callable_method_filter(attr):
             )
         if self.is_function:
             return self.callable
+
         return getattr(self.callable, method_name)
 
-    async def send_user_result_over_asgi(self, result: Any, asgi_sender: Send):
+    async def send_user_result_over_asgi(
+        self, result: Any, scope: Scope, asgi_sender: Send, asgi_receive: Receive
```

Comment:
[Nit] The code uses both `asgi_sender` (noun) and `asgi_receive` (verb). Could we pick one tense and keep it consistent?

Reply:
yes I'm going to rename these to asgi standard names everywhere (just `send` and `receive`)

Reply:
updated